README.md

```diff
@@ -34,29 +33,23 @@ To setup the project locally read these wiki pages and follow the instructions:
  - [Fork, Clone and Remote](https://github.com/anitab-org/mentorship-backend/wiki/Fork%2C-Clone-%26-Remote)
  - [Export Environment Variables](docs/environment-variables.md)
 
-### Run app in Windows
+### Run app locally
 
-The project runs on Python 3.
+The project runs on Python 3.9. We use [Poetry](https://python-poetry.org/) to manage dependencies. It's powerful yet simple to use.
+Here are a few basic commands you'll use the most:
+- `poetry install` - installs the dependencies from `pyproject.toml` package
+- `poetry add <package>` - adds the `<package>` to `pyproject.toml` and installs it
+- `poetry remove <package>`- removes the `<package>` from `pyproject.toml` and uninstalls it
+- `poetry shell` - spawns a shell within the virtual environment. Equivalent of `virtualenv activate`, for those familiar with [`virtualenv`](https://pypi.org/project/virtualenv/)
+- `deactivate` - deactivates the virtual environemnt. Counterpart for `poetry shell`. Use this when you finish work.
```

Comment:
could you put these in a subsection about managing dependencies perhaps, it looks confusing being just together with the steps. being in a separate and clear section would be easier to skim through.

Reply:
I redesigned this section a bit. Let me know what you think:)

Reply:
Looks good :) I just added a comment about "improving" the text that makes the intro to the poetry commands.